src/data/real/ennreal.lean

```diff
@@ -4,6 +4,7 @@ Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Johannes Hölzl, Yury Kudryashov
 -/
 import data.real.nnreal
+import order.liminf_limsup
```

Comment:
Perhaps these lemmas would fit better in `topology/instances/ennreal`? I think it wouldn't require additional imports there.

Reply:
That makes sense. I was having trouble figuring out where those lemmas should go and I didn't like the extra import.